docs/release-source/release/fakes.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: Fakes - Sinon.JS
+breadcrumb: fakes
+---
+
+### Introduction
+
+Fakes are a new concept to Sinon, simplifying and merging the concepts from `spies` and `stubs`.
+
+A test fake is a `Function` that records arguments, return value, the value of
+`this` and exception thrown (if any) for all its calls.
+
+Fakes can be created with or without behaviour.
+
+All fakes are immutable: once created, their behaviour will not change.
+
+Unlike `sinon.spy` and `sinon.stub` APIs, the `sinon.fake` API knows only how to create fakes, and doesn't concern itself with plugging them into the system under test. To plug the fakes into the system under test, you can use the [`sinon.replace*`](../sandbox#replace) methods.
+
+
+### Creating a fake
+
+```js
+// create a basic fake, with no behaviour
+var fake = sinon.fake();
+
+fake();
+
+console.log(fake.callCount);
+// 1
+```
+
+### Fakes with behaviour
+
+Fakes can be created with behaviour, which cannot be changed once the fake has been created.
```

Comment:
IMO it would be good to point out that it is possible to do [a `restore` as mentioned here](https://github.com/sinonjs/sinon/pull/1586/files#diff-8e4eec829308104e4aa45dca14337830R116) and that would be enough for the user to get the original property back into the object.

As a user I think that would be something I'd like to know how to do as soon as I read that:

> [a fake] cannot be changed once the fake has been created

Reply:
`replace` and `restore` belong together. Fakes are created separately from replacing properties. 

Fakes are just one possible argument to `replace`, you could use something else if you wanted to (like `mock`, `spy` or `stub` ... or even `Date`).

I'd like to keep the documentation focused, so not mention topics that are not necessary for understanding the current topic. With that in mind, I think that mentioning `replace` in the documentation about creating fakes with behaviour, would reduce the focus and make the text a bit confusing.

Reply:
I totally understand, but then maybe we could rephrase this to make it more specific.

The first time I read this I had the strong impression that you would not be able to restore any property replaced with a fake.

The wording is a bit difficult though, what do y'all think? 🤔

Reply:
@lucasfcosta I think the documentation already points out that `replace` is a sandbox method that _may_ use a fake as an argument. The standbox can then be restored. There is nothing to restore in the fake itself. I agree with @mroderick here that talking about `restore` here would make the documentation more confusing.